### PR TITLE
New landscapes

### PR DESCRIPTION
**Pull request by jafdy**
The result of this contest.
http://steamcommunity.com/app/404410/discussions/3/144513524085112554/
2 new images that I have the copyright for and 4 more images that will be mountains 22-25 when I get hold of the photographer.

### CODE REVIEW

**Comment by jafdy** [comment]
I got fed up with waiting for the guy to get online. Permition to use was given when he entered the contest. @endless-sky I think this is ready to be merged if you are happy with it.

**Comment by endless-sky** [comment]
Do you have 2x versions of these?

Also, the horizon on the second city image is tilted by several degrees, enough that it's pretty noticeable. If the source image has high enough resolution, you should rotate it to level it out.

**Comment by jafdy** [comment]
I do have the 2x versions. They where the ones I uploaded originally before I corrected the size.
I'll get that horizon sorted out.

File: copyright [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+﻿Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```
Comment (endless-sky) [inline copyright]: It looks like your editor inserted an invisible byte order mark here, which is completely unnecessary for UTF-8 files and will confuse the heck out of certain other text editors.

Comment (jafdy) [inline copyright]: Umm...OK. I don't know what that means. I'll try to get rid of it.

Comment (jafdy) [inline copyright]: Got it.

**Comment by Bladewood** [comment]
"bite"

also, yeah, it works.

**Pull request merged**